src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -5142,6 +5143,29 @@ .maxstack  7
 ");
         }
 
+        [Theory, WorkItem(48919, "https://github.com/dotnet/roslyn/issues/48919")]
+        [InlineData("==")]
+        [InlineData("!=")]
+        [InlineData(">=")]
+        [InlineData(">")]
+        [InlineData("<=")]
+        [InlineData("<")]
+        public void BinaryComparisonWarnings(string @operator)
```

Comment:
Just want to check: comparisons to `null` remain valid and tested correct?

Reply:
Yep, we have multiple tests covering comparisons to null, both in patterns and with equality operators.